snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/ClosedChannel.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.kotlinmpplib
+
+import breez_sdk.*
+
+class ClosedChannel {
+    func prepareSweep(sdk: BlockingBreezServices, feeRate: int) {
```

Comment:
I think uint64 is ULong
```suggestion
    func prepareSweep(sdk: BlockingBreezServices, feeRate: ULong) {
```